pyproject.toml

```diff
@@ -28,15 +28,15 @@ classifiers = [
 python = "^3.7"
 
 [tool.poetry.dev-dependencies]
-numpy = "1.21"
-tox = "^3.24.1"
+numpy = "^1.21"
```

Comment:
If arm runs tiny build, do we need the modifications in this file and the requirements file?

Reply:
No, but why not upgrade?